### PR TITLE
Don't complain about MKL threading on mac since openmp doesn't work.

### PR DESCRIPTION
**Pull request by abergeron**

### CODE REVIEW

File: theano/configdefaults.py [hunk 1]
```diff
@@ -1241,7 +1241,9 @@ def is_valid_check_preallocated_output_param(param):
 def check_mkl_openmp():
     if not theano.config.blas.check_openmp:
         return
-    import os
+    import os, sys
```
Comment (nouiz) [inline theano/configdefaults.py]: why the import was here and why you extend it? Both of those import are already at the top of this file.

Comment (abergeron) [reply theano/configdefaults.py]: I don't remember why it was there, but I extended it to be sure.

File: theano/configdefaults.py [hunk 2]
```diff
@@ -1241,7 +1241,8 @@ def is_valid_check_preallocated_output_param(param):
 def check_mkl_openmp():
     if not theano.config.blas.check_openmp:
         return
-    import os
+    if sys.platform == 'darwin':
```
Comment (nouiz) [inline theano/configdefaults.py]: Could we also check if this is a conda python or not? I don't think we have this logic

Comment (abergeron) [reply theano/configdefaults.py]: We don't need that.

clang doesn't support OpenMP so even if you're not in conda you still won't have problems.

Or did you have something else in mind?

**Pull request merged**